src/builder/build_traits.rs

```diff
@@ -301,10 +300,16 @@ pub trait Dataflow: Container {
     /// This function will return an error if there is an error when adding the node.
     fn load_const(&mut self, cid: &ConstID) -> Result<Wire, BuildError> {
         let const_node = cid.node();
+        let op @ ops::Const { .. } = self
```

Comment:
(this is the NodeID::get_node from above)

Reply:
Why `op @ ops::Const { .. }` rather than `op: ops::Const` ?

Reply:
vestigial, thanks